examples/nlp/token_classification/token_classification.py

```diff
@@ -48,7 +48,7 @@
 parser.add_argument("--ignore_start_end", action='store_false')
 parser.add_argument("--ignore_extra_tokens", action='store_false')
 parser.add_argument("--none_label", default='O', type=str)
-parser.add_argument("--do_not_shuffle_data", action='store_false')
+parser.add_argument("--shuffle_data", action='store_true')
```

Comment:
update this to recent PR

Reply:
Isn't it the right way? Which PR?

Reply:
why do you want to change this line? we had this discussion here already in https://github.com/NVIDIA/NeMo/pull/401

Reply:
If you set the default to store_false, then when use --shuffle_data it would disable the shuffling. The default for such args is False and I am not sure how to have True as default. Do you suggest to change the name to do_not_shuffle_data?

Reply:
yes, basically the outcome of #401 is to use --do_not_shuffle_data", action='store_false'. This will by default be true, and only false if you explicitly pass --do_not_shuffle_data

Reply:
Changed it to :
parser.add_argument("--no_shuffle_data", action='store_false', dest="shuffle_data")